LayoutTests/fast/css/svg-currentcolor-dynamic-inherit.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<head>
+<style>
+input:checked ~ div > svg {
+   color: green;
+}
+svg {
+   width: 10%;
+}
+</style>
+</head>
+<body>
+
+<script src="/resources/testdriver.js"></script>
```

Comment:
You can't use absolute URLs in tests in `fast`

Reply:
Fixed